Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -202,16 +202,6 @@ export class BotProject implements IBotProject {
   public getEnvSettings = async (obfuscate: boolean) => {
     const settings = await this.settingManager.get(obfuscate);
 
-    // Resolve relative path for custom runtime if the path is relative
-    if (settings?.runtime?.customRuntime && settings.runtime.path && !Path.isAbsolute(settings.runtime.path)) {
-      const absolutePath = Path.resolve(this.dir, 'settings', settings.runtime.path);
-
-      if (fs.existsSync(absolutePath)) {
-        settings.runtime.path = absolutePath;
-        await this.updateEnvSettings(settings);
-      }
-    }
```

Comment:
Why was this code added in the first place? Will we have a regression if we remove it?

Reply:
This was incorrectly added during an earlier part of the creation work. It was introduced after the r11 release.